docs/git-compatibility.md

```diff
@@ -196,8 +215,8 @@ jj new && jj undo
     Instead of the `echo -n ...` line, use:
     `Set-Content -Path .jj/repo/store/git_target -Value ../../../.git -NoNewLine`
 
-We may officially support this in the future. If you try this, we would
-appreciate feedback and bug reports.
+    We may officially support this in the future. If you try this, we would
+    appreciate feedback and bug reports.
```

Comment:
nit: I think you can delete the paragraphs above this, since there's a command now. I only noticed it because the file somehow got auto-formatted

Reply:
I've removed that command but I have kept the description of the manual process in case somebody wants to know how it works. Maybe once the command has been included in jj for a version or two we could remove that as well.